pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/ActivityInstance.java

```diff
@@ -146,4 +157,12 @@ public boolean isInstanceHidden() {
     public boolean isHidden() {
         return isHidden;
     }
+
+    public String getParentInstanceGuid() {
+        return parentInstanceGuid;
+    }
+
+    public void setParentInstanceGuid(String parentInstanceGuid) {
+        this.parentInstanceGuid = parentInstanceGuid;
```

Comment:
Do we have to worry that this is only property in object whose value is not set in constructor? Would be more consistent if if treated like others.

(In ActivityInstanceSummary is almost same deal, but looks to me like code for setters here was added before?)

Reply:
I think this was from a merge conflict. I have resolved conflicts and don't see this change in this PR.